### PR TITLE
Added vidlit host.

### PR DESCRIPTION
**Pull request by billmalarky**
Hello honestbleeps. I'd like to add a video project of mine to RES.

I went ahead and ran eslint and followed your style guidelines. Everything seems to be working great.

Please let me know if I'm forgetting anything!

### CODE REVIEW

**Comment by billmalarky** [comment]
@erikdesjardins Is this PR good to go or is there something else I need before it can be merged?

Thank you.

**Comment by erikdesjardins** [comment]
Sorry about the delay; this is now slightly out of date, I'll merge it immediately after it's updated.

File: lib/modules/hosts/vidlit.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (erikdesjardins) [inline lib/modules/hosts/vidlit.js]: Nit: no newline at start of file

File: lib/modules/hosts/vidlit.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { ajax } from '../../environment';
+
+export default {
+	moduleID: 'vidlit',
+	name: 'vidlit',
+	domains: ['vidl.it'],
+	logo: 'https://vidl.it/favicon.ico',
+	detect: href => (/^https?:\/\/vidl.it\/([A-Za-z0-9]+)$/i).exec(href),
```
Comment (erikdesjardins) [inline lib/modules/hosts/vidlit.js]: `detect: href =>` -> `detect: ({ href }) =>`

File: lib/modules/hosts/vidlit.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	async handleLink(elem, [, hash]) {
```
Comment (erikdesjardins) [inline lib/modules/hosts/vidlit.js]: `elem` -> `href`

File: lib/modules/hosts/vidlit.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		const meta = await ajax({
+			url: `https://vidl.it/embed/meta/${hash}`,
+			type: 'json',
+		});
+		elem.type = 'IFRAME';
```
Comment (erikdesjardins) [inline lib/modules/hosts/vidlit.js]:
```js
return {
  type: 'IFRAME',
  embed: `https://vidl.it/embed/${hash}`,
  width: `${meta.data.iframe.width}px`,
  height: `${meta.data.iframe.height}px`,
};
```

File: lib/modules/hosts/vidlit.js (hunk 1 above)
Comment (billmalarky) [inline lib/modules/hosts/vidlit.js]: Feel free to nitpick! I want this to comply with the style guidelines.

**Comment by billmalarky** [comment]
@erikdesjardins really appreciate your detailed feedback, made updating this super easy. Let me know if it is missing anything else.

**Comment by erikdesjardins** [comment]
Looks good, thanks!

**Pull request merged**